cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java

```diff
@@ -81,13 +81,14 @@
                         MessagingService messagingService,
                         @Named(REMOTE_TETHERING_AUTHENTICATOR) RemoteAuthenticator remoteAuthenticator) {
     super(RetryStrategies.fromConfiguration(cConf, "tethering.agent."));
-    this.connectionInterval = TimeUnit.SECONDS.toMillis(cConf.getLong(Constants.Tethering.CONNECTION_INTERVAL));
+    this.connectionInterval = TimeUnit.SECONDS.toMillis(cConf.getLong(Constants.Tethering.CONNECTION_INTERVAL, 10L));
     this.cConf = cConf;
     this.transactionRunner = transactionRunner;
     this.store = store;
     this.instanceName = cConf.get(Constants.INSTANCE_NAME);
     this.lastMessageIds = new HashMap<>();
-    this.topic = new TopicId(NamespaceId.SYSTEM.getNamespace(), cConf.get(Constants.Tethering.TETHERING_TOPIC));
+    this.topic = new TopicId(NamespaceId.SYSTEM.getNamespace(), cConf.get(Constants.Tethering.TETHERING_TOPIC,
+                                                                          "tethering"));
```

Comment:
This conflicts with https://github.com/cdapio/cdap/pull/14050. I'll update this PR once https://github.com/cdapio/cdap/pull/14050 is merged

Reply:
merged the PR!